content/blog/2020-06-22-dvc-1-0-release.md

```diff
@@ -62,21 +62,21 @@ learned in 3 years of this journey and how these are reflected in the new DVC.
 
 Our users taught us that ML pipelines evolve much faster than data engineering
 pipelines with data processing steps. People need to change the commands of the
-pipeline often and it was not easy to do with the old DVC files.
+pipeline often and it was not easy to do this with the old DVC-files.
 
-In DVC 1.0, the DVC file format was changed in three big ways. First, instead of
-multiple DVC stage files (`*.dvc`), each project has a single DVC file
-`dvc.yaml`. By default, all stages go in this single `.yaml` file.
+In DVC 1.0, the DVC metafile format was changed in three big ways. First,
+instead of multiple DVC stage files (`*.dvc`), each project has a single
+`dvc.yaml` file. By default, all stages go in this single YAML file.
```

Comment:
> each project has a single `dvc.yaml` file

Actually you can have a dvc.yaml file in each subdirectory if you wanted. At least right now it's possible. Should that be mentioned @dmpetrov ?